source/com.microsoft.tfs.jni/src/com/microsoft/tfs/jni/helpers/LocalHost.java

```diff
@@ -108,10 +110,77 @@ public synchronized static String getShortName() {
                 name.substring(0, (name.length() > MAX_COMPUTER_NAME_SIZE) ? MAX_COMPUTER_NAME_SIZE : name.length());
         }
 
+        log.info("Short name resolved to: " + computerNameJB); //$NON-NLS-1$
         return computerName;
     }
 
     /**
+     * <p>
+     * Gets the short name of the current computer compatible with JetBrains TFS
+     * plugin.
+     * <p>
+     * The only difference with the traditional TEE method
+     * {@link #getShortName()} is in the order of particular name detection
+     * sub-method calls. This method prefers {@link #getPureJavaShortName()} to
+     * {@link #getNativeShortName()}
+     * </p>
+     * <p>
+     * The {@link #SHORT_NAME_OVERRIDE_PROPERTY} may be used to define the
+     * computer name exactly.
+     * </p>
+     *
+     * @return a short identifier (name of this computer). Never
+     *         <code>null</code> or empty.
+     */
+    public synchronized static String getShortNameJB() {
```

Comment:
Use the same way to figure out the short name like JetBrains
https://github.com/JetBrains/tfsIntegration/blob/0f127c1b10acd3f985920246f43dfbe170d69960/src/org/jetbrains/tfsIntegration/core/tfs/Workstation.java#L234